packages/mobile/src/identity/actions.ts

```diff
@@ -31,6 +31,7 @@ export enum Actions {
   VALIDATE_RECIPIENT_ADDRESS = 'SEND/VALIDATE_RECIPIENT_ADDRESS',
   VALIDATE_RECIPIENT_ADDRESS_SUCCESS = 'SEND/VALIDATE_RECIPIENT_ADDRESS_SUCCESS',
   REQUIRE_SECURE_SEND = 'SEND/REQUIRE_SECURE_SEND',
+  DONE_FETCHING_ADDRESSES = 'DONE_FETCHING_ADDRESSES',
```

Comment:
An `END_FETCHING_ADDRESSES` or a `SET_FETCH_ADDRESS_STATUS` would be more consistent with the other status-tracking actions we use here and elsewhere

Reply:
And it would be nice to get success/fail status included in it so the UI can take that into account

Reply:
Yeah, I think I'll change the naming to `END_FETCHING_ADDRESSES`. My intention was for this to purely signal the end of a fetch (whether successful or not), not the status.

Any failure in the flow is currently being caught and shown to the user as an `ADDRESS_LOOKUP_FAILURE`. Do you think there is some other success/fail status that may be useful?

Reply:
Possibly, but if you don't have an immediate need for it we probably don't need to block this PR for it :)